### PR TITLE
Filter added for conference and division

### PR DESCRIPTION
**Pull request by miqdadhafiz35**
Added conference filter and division filter 
CSS edited to make it look a bit nicer but needs work still

### CODE REVIEW

File: views/home.ejs [hunk 1]
```diff
@@ -5,14 +5,37 @@
 		<a href="/"><img src="/images/nba-logo.svg" alt="NBA logo" id="logoNBA" /></a>
     </div>
 
+
   <div class="container">
 	  <div class="container-fluid">
+
+	<div id="filterc">
+	<select class="selectpicker form-control" id="filterC" aria-label="Conference">
+	<option selected disabled>Conference menu</option>
```
Comment (muhtacinm) [inline views/home.ejs]: Maybe best to leave the word "menu" out.

File: views/home.ejs [hunk 2]
```diff
@@ -5,14 +5,37 @@
...
+	<option value="1">Eastern Conference</option>
+	<option value="2">Western Conference</option>
+	</select>
+	</div>
+
+	<div id="filterd">
+	<select class="selectpicker form-control" id="filterD" aria-label="Divisions">
+	<option selected disabled>Divisions</option>
```
Comment (muhtacinm) [inline views/home.ejs]: Maybe best to rename to "Division" to follow same convention as "Conference"

Comment (muhtacinm) [reply views/home.ejs]: Would also be nice if you could center the text in the select options

File: views/home.ejs (hunk 1 above)
Comment (muhtacinm) [reply views/home.ejs]: Would also be nice if you could center the text in the select options

File: public/css/style.css [hunk 3]
```diff
@@ -90,3 +91,17 @@
...
+    width:15%;
+    padding-left:10px;
+}
+#filterc {
+    float: left;
+    clear: none;
+    padding-left:450px;
+    display:flex;
+    margin:auto;
+}
```
Comment (muhtacinm) [inline public/css/style.css]: image

Mobile view looks off

File: public/js/main.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    var value = $('#filterC :selected').text();
+    $('.teamCard').hide();
+    $('[data-conference="' + value + '"]').show();
+});
+
+$('#filterD').change(function () {
+    var value1 = $('#filterD :selected').text();
+    $('.teamDiv').hide();
+    $('[data-division="' + value1 + '"]').show();
+});
```
Comment (muhtacinm) [inline public/js/main.js]: Add comments to specify what functions are doing.

File: views/home.ejs [hunk 5]
```diff
@@ -5,14 +5,37 @@
 		<a href="/"><img src="/images/nba-logo.svg" alt="NBA logo" id="logoNBA" /></a>
     </div>
 
+
   <div class="container">
 	  <div class="container-fluid">
```
Comment (muhtacinm) [inline views/home.ejs:10]: image

Maybe add something to indicate these are filters

**Review by muhtacinm — APPROVED** [review]

**Review by dettloffd — APPROVED** [review]

**Pull request merged**